builder/index.html

```diff
@@ -394,16 +391,15 @@ <h4 class="panel-title">
                             </li>
                         </ul>
                     </div>
-                    <div class="col-md-3" class="bs-sidebar navbar" >
-                        <ul class="nav bs-sidenav" style="position:fixed; margin-top:68px;">
+                    <div class="col-lg-2 col-md-2 col-sm-2 col-xs-4 ncf_col" class="bs-sidebar navbar" ng-if="selectedMethod === undefined">
```

Comment:
This is what happens when I set col-lg-1 to the "Categories" list :

![col-lg-1](https://cloud.githubusercontent.com/assets/9928447/18837978/0a765a58-8406-11e6-9610-16c49600a3e2.png)

It works properly on very large screen ( width > ~1450px), but below this width, some categories names are larger than the column width :/